src/pages/home/report/ReportActionCompose.js

```diff
@@ -196,7 +197,6 @@ class ReportActionCompose extends React.Component {
 
     /**
      * Updates the Highlight state of the composer
-     *
```

Comment:
Please undo these unrelated changes to reduce the diff of this PR. This will make the PR easier to review, reduce the likelihood of conflicts, simplify the git blame, etc... Plus people have different code style preferences so unless something is prescribed in our [official style guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md) code style changes are probably not worth making. Thanks for understanding!

Reply:
Done.